freqtrade/edge/edge_positioning.py

```diff
@@ -399,9 +400,8 @@ def _detect_next_stop_or_sell_point(self, buy_column, sell_column, date_column,
                 # trade opens in reality on the next candle
                 open_trade_index += 1
 
-            stop_price_percentage = stoploss + 1
             open_price = ohlc_columns[open_trade_index, 0]
-            stop_price = (open_price * stop_price_percentage)
+            stop_price = (open_price * (stoploss + 1))
```

Comment:
@xmatthias pls review this separately (not related to the changes in this PR, a separate issue): stoploss is used here without `abs`...

Reply:
not using `abs()` is fine in theory if we're sure the value is negative ... 

in this case however, the source for this stoploss is the loop over stoploss_range - and i currently struggle to find any code that ensures `stoploss_range_min` and `stoploss_range_max` must be negative (it's not in the config-validation either) - so i suspect you could run this with positive values - which would produce rather odd results ...

i'd leave that for another PR though ... not "really" related to this if we're honest ...